docs/sources/introduction/_index.md

```diff
@@ -6,10 +6,6 @@ weight = 5
 
 # Introduction to Grafana
 
-Grafana is a complete observability stack that allows you to monitor and analyze metrics, logs and traces. It allows you to query, visualize, alert on and understand your data no matter where it is stored. Create, explore, and share beautiful dashboards with your team and foster a data driven culture. For more information, refer to [Grafana overview](https://grafana.com/grafana/). Our observability stack has the following products and components.
+Grafana is a complete observability stack that allows you to monitor and analyze metrics, logs and traces. It allows you to query, visualize, alert on and understand your data no matter where it is stored.
 
-{{< docs/shared "basics/what-is-grafana.md" >}}
-
-{{< docs/shared "basics/grafana-cloud.md" >}}
-
-{{< docs/shared "basics/grafana-enterprise.md" >}}
```

Comment:
this will make this index page kind of empty and pointless, not even any links to relevant info.

Reply:
Added the Grafana OSS content to the index page so it has substance.